### PR TITLE
Scrs 4520 ct active date

### PR DESCRIPTION
**Pull request by PeterPerhac**

### CODE REVIEW

File: app/services/IncorporationInformationService.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@ImplementedBy(classOf[IncorporationInformationService])
+trait IIService {
+
+  def getCTActiveDate()(implicit headerCarrier: HeaderCarrier): OptionalResponse[LocalDate]
+
+}
+
+class IncorporationInformationService @Inject()(ctConnector: PPConnector) extends IIService with CommonService {
```
Comment (peter-hazell) [inline app/services/IncorporationInformationService.scala]: Let's rename this as currently it is a bit mis-leading, this is not the actual IIService, just a central service within vat registration for us to get pieces of information about the company, i.e. CT active date

File: app/connectors/CompanyRegistrationConnector.scala [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  def getCompanyRegistrationDetails
+  (regId: String)
+  (implicit hc: HeaderCarrier, rds: HttpReads[CorporationTaxRegistration]): OptionalResponse[CorporationTaxRegistration]
+
+}
+
+
+@Singleton
+class CompanyRegistrationConnector extends PPConnector with ServicesConfig {
```
Comment (peter-hazell) [inline app/connectors/CompanyRegistrationConnector.scala]: Let's rename this as it is Pre pop connector, not Company reg connector

File: conf/messages [hunk 3]
```diff
@@ -44,7 +44,7 @@ pages.startDate.title                                 = Start Date
 pages.startDate.heading                               = What do you want your VAT start date to be?
 pages.startDate.para1                                 = You can choose the date the company becomes VAT registered when you register voluntarily.
 pages.startDate.radio.whenRegistered                  = The date the company is registered with Companies House
-pages.startDate.radio.whenTrading                     = The same date you start doing business
```
Comment (peter-hazell) [inline conf/messages]: This should be: 'The date the company starts doing business:'

Comment (PeterPerhac) [inline conf/messages]: done

File: app/connectors/CompanyRegistrationConnector.scala (hunk 2 above)
Comment (PeterPerhac) [inline app/connectors/CompanyRegistrationConnector.scala]: done

File: app/services/IncorporationInformationService.scala (hunk 1 above)
Comment (PeterPerhac) [inline app/services/IncorporationInformationService.scala]: i'll change to prepopulation service

**Pull request merged**